tools/docker/README.md

```diff
@@ -3,31 +3,44 @@ From the root directory of the ockam codebase:
 
 ## Base
 
+- This image has things all container images will need - entrypoint, locales, curl.
+- It is based on a lightweight version of [Debian](https://hub.docker.com/_/debian).
+- It has no build tools.
+- It is designed to be the base of images that are running in production and don’t need build tools.
+
+Build the base:
+
 ```
 docker build \
-  --build-arg BASE_IMAGE=debian:10.7-slim@sha256:240f770008bdc538fecc8d3fa7a32a533eac55c14cbc56a9a8a6f7d741b47e33 \
```

Comment:
Is this command line option truly not needed? If it's not, can it be omitted from `ockam/builder_base` below too?

Reply:
Its updated version is already specified in the [Dockerfile](https://github.com/ockam-network/ockam/blob/5e8af98de33949e057d04848afbca4587ed2f23c/tools/docker/base/Dockerfile#L1), do we still need to pass it as an argument?

Reply:
Right, but the image and hash are already specified in an `ARG` statement in the Dockerfile. Just making sure we want the redundancy here.

Reply:
Added --build-arg